<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>760</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>690</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zamknij</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>831</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutUp">
     <item>
      <widget class="QLabel" name="labelAppTitle">
       <property name="font">
        <font>
         <family>Yu Gothic</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>APProduct</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAppDesc">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>APProduct - aplikacja, która pozwoli Ci świadomie wybrać produkty na zakupach! :)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>331</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutPhoto">
     <item>
      <widget class="QLabel" name="labelPhoto">
       <property name="text">
        <string>Zdjęcie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPhoto_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutIngredients">
     <item>
      <widget class="QLabel" name="labelIngredients">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Skład</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableViewIngredients"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutCalorieChart">
     <item>
      <widget class="QLabel" name="labelCalorieChart">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wartość odżywcza</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableViewCalorieChart"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelDB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[+]Status</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewCategories">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNabial">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Nabiał</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNapoje">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Napoje</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMakarony">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Makarony</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSlodycze">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Słodycze</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewProduct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuestion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>O którym produkcie chcesz dowiedzieć się więcej? Wpisz jego ID poniżej.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelCategories">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kategorie:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProducts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Produkty:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOK">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>631</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
